sr_robot_launch/config/gazebo/controller/la_group_position_controller.yaml

```diff
@@ -0,0 +1,11 @@
+la_group_position_controller:
+  type: "position_controllers/JointGroupPositionController"
+  joints:
+    - la_shoulder_pan_joint
+    - la_shoulder_lift_joint
+    - la_elbow_joint
+    - la_wrist_1_joint
+    - la_wrist_2_joint
+    - la_wrist_3_joint
+    - rh_WRJ1
```

Comment:
should be lh for the wrist

Reply:
Fixed in https://github.com/shadow-robot/sr_interface/pull/419/commits/69bc13be8e64f769c2179bda3d2bfc388b4bc071